### PR TITLE
Stand tests update

### PR DESCRIPTION
**Pull request by awirb**
* update of benchmarks regarding the new simTypes, works for com1DFAPy for now
* TODO adapt com1DFA simulation naming - Done with newest commit 
* also updated variations test 
* now com1DFA also uses simTypes ent, res, entres, null and available

### CODE REVIEW

**Review by fso42 — APPROVED** [review]

File: avaframe/com1DFA/com1DFA.py [hunk 1]
```diff
@@ -93,6 +93,71 @@ def copyReplace(origFile, workFile, searchString, replString):
         file.write(fileData)
 
 
+def getSimulation(cfg, rel, entResInfo):
+    """ get Simulation to run for a given release
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:97]: Bit unclear what is meant here. Maybe 
Given a simulation scenario, returns the types of simulations that need to be performed

File: avaframe/com1DFA/com1DFACfg.ini [hunk 2]
```diff
@@ -8,10 +8,12 @@
...
 # True if full output shall be printed to terminal
 fullOut = False
-# True if entrainment areas shall NOT be included
-noEntrainment = False
-# True if resistance areas shall NOT be included
-noResistance = False
+
+#++++++++++++++++ simulation type
+# list of simulations that shall be performed (null, ent, res, entres, available (use all available input data))
+simTypeList = null|available
```
Comment (fso42) [inline avaframe/com1DFA/com1DFACfg.ini:14]: should be only `available`, since `available` contains `null`...

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/com1DFA.py (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py:97]: I changed this to: Get a list of all simulations that shall be performed according to simTypeList in configuration file;
        and a dictionary with information on release area

File: avaframe/com1DFA/com1DFACfg.ini (hunk 2 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFACfg.ini:14]: correct, set it to available

**Review by fso42 — APPROVED** [review]

**Pull request merged**